### PR TITLE
Feat/add tax calculators resources

### PR DESCRIPTION
**Pull request by incentrotolgaakyazi**

### CODE REVIEW

File: commercelayer/resource_manual_tax_calculator.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+		ReadContext:   resourceManualTaxCalculatorReadFunc,
+		CreateContext: resourceManualTaxCalculatorCreateFunc,
+		UpdateContext: resourceManualTaxCalculatorUpdateFunc,
+		DeleteContext: resourceManualTaxCalculatorDeleteFunc,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Description: "The manual payment unique identifier",
```
Comment (demeyerthom) [inline commercelayer/resource_manual_tax_calculator.go:23]: I think you need to change the description here

File: commercelayer/resource_manual_tax_calculator.go [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+								Type: schema.TypeString,
+							},
+							Optional: true,
+						},
+					},
+				},
+			},
+		},
+	}
+}
```
Comment (demeyerthom) [inline commercelayer/resource_manual_tax_calculator.go:71]: I am assuming the tax rules relation will follow in the follow-up pr about the tax rules?

File: commercelayer/resource_taxjar_account.go [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+							Optional: true,
+						},
+					},
+				},
+			},
+		},
+	}
```
Comment (demeyerthom) [inline commercelayer/resource_taxjar_account.go:75]: Same as above, will the relations be in the follow-up?

**Review by demeyerthom — CHANGES_REQUESTED** [review]
Very nice! Don't forget to also generate the docs!

File: commercelayer/resource_taxjar_account.go (hunk 3 above)
Comment (incentrotolgaakyazi) [reply commercelayer/resource_taxjar_account.go:75]: That's the plan indeed :)

**Review by demeyerthom — APPROVED** [review]

**Pull request merged**